mcp/housekeeping/labjack_functions.c

```diff
@@ -536,7 +540,9 @@ void heater_write(int m_labjack, int address, float command) {
             }
         }
     }
-    if (m_labjack == 1 && address > 2008) {
+}
+// old BLAST code that would read certain registers on LABJACK CRYO 2 because we used them as inputs
```

Comment:
Is there a chance this ever comes back? Or can it be deleted?

Reply:
It may come back if anyone wants to use the digital I/O as a "status" bit where you don't or can't use the AIN. I'd like to keep the reference around.